assets/ts/app/global-navigation.ts

```diff
@@ -307,9 +310,29 @@ export function setup(rootElement: HTMLElement): void {
     }
   }
 
+  function refocusLastActiveMenu(): void {
+    if (activeMenuElement && activeMenuElement instanceof HTMLElement) {
+      activeMenuElement.focus();
+    }
+  }
+
+  // Save the last focused menu toggle
+  var activeMenuElement: Element | null = null;
+  function saveActiveMenuElement(): void {
+    if (document.activeElement?.className.includes("m-menu--desktop__toggle")) {
+      activeMenuElement = document.activeElement;
+    } else if (
+      document.activeElement?.className.includes("custom-language-selector")
```

Comment:
This is kind of hacky, and I am not a huge fan.  I was trying to come up with a solution to clear the last active menu element when the user tab leaves the menu buttons.  Many of the sub menu options don't have classes so it was hard to tell where we were in the tree when they were tab selected.
The next item hit after leaving the menu buttons are the language translation button, so I decided to clear it then.
Open to other ideas.